crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Generic.hs

```diff
@@ -689,7 +689,7 @@ readMem sym w gsym l tp alignment m = do
             return $ Partial.totalLLVMVal sym $ LLVMValInt blk0 byte
       genValueCtor sym (memEndianForm m) mop
         =<< loadTypedValueFromBytes 0 tp loadArrayByteFn
-    Nothing -> readMem' sym w (memEndianForm m) gsym l m tp alignment (memWrites m)
+    _ -> readMem' sym w (memEndianForm m) gsym l m tp alignment (memWrites m)
```

Comment:
is it possible to keep `Nothing`? I don't like matching on `_`

Reply:
No, because the previous `Just` clause has a guard.  We could add an `otherwise` and repeat the RHS, but I think that's worse than a wildcard match.